src/components/SlidePanel/SlidePanel.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import "./_slide-panel.scss";
+
+function SlidePanel({ children, onClose }) {
+  return (
+    <div className="slide-panel">
+      <div className="slide-panel--close">
```

Comment:
`slide-panel__close` this is an element of the slide-panel not a modifier to it.

Reply:
removed this in favour of your other comment.